nsv13/code/modules/overmap/physics2d.dm

```diff
@@ -83,6 +83,11 @@ PROCESSING_SUBSYSTEM_DEF(physics_processing)
 		var/obj/item/projectile/P = holder
 		if(istype(P))
 			P.physics2d = null
+	for(var/I in SSphysics_processing.physics_levels)
+		var/list/za_warudo = SSphysics_processing.physics_levels[I]
+		for(var/datum/component/physicsbody in za_warudo)
```

Comment:
I think `if (find(src))` works and is cheaper? Ask coderbus, but this feels horribly expensive and jank, and you're looping over all active physics objects regardless of if you find yourself in literally the first object of the first level.